### PR TITLE
add damage parameters for tropical cyclone damage function

### PR DESCRIPTION
**Pull request by fpiontek**
This adds the data in TCdamageKrichene to the REMIND input data. There are two subtypes for two damage parameters.

### CODE REVIEW

**Review by LaviniaBaumstark — COMMENTED** [review]
looks good, please ensure that the checks do not fail

File: R/convertTCdamage.R [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#' Convert TCdamage
+#' fills in countries not affected by tropical cyclones (TC), setting parameters to zero
+#' @author Franziska Piontek
+#' @param x is MAgPIE object containing the damage parameters for the TC-prone countries
+#' @return MAgPIE object containing values for all 249 ISO countries
+
+convertTCdamage <- function(x){
+	out <- toolCountryFill(x,fill=0,verbosity=3)
```
[the remark this answers is not in the document]
Comment (fpiontek) [reply R/convertTCdamage.R]: I changed that and fixed the typo which caused the check to fail.

File: R/calcTCdamage.R [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#' write TC damage parameters into input data
+#' they are country-specific and should not be aggregated to the regional level at all
+
+#' @author Franziska Piontek
+#' @return MAgPIE object of damage parameters for country level tropical cyclone damage function
+
```
Comment (LaviniaBaumstark) [inline R/calcTCdamage.R:6]: you have to document the subtype here

**Review by LaviniaBaumstark — CHANGES_REQUESTED** [review]

File: R/calcTCdamage.R (hunk 2 above)
Comment (fpiontek) [reply R/calcTCdamage.R:7]: done

**Review by LaviniaBaumstark — APPROVED** [review]

**Pull request merged**